rca/scholarships/tests/test_models.py

```diff
@@ -31,16 +31,14 @@ def test_factories(self):
 
 
 class TestScholarshipsListingPageRules(WagtailPageTests):
-    def setUp(self):
-        self.home_page = HomePage.objects.first()
```

Comment:
This query runs for a test that doesn't need it

Reply:
yea it was used in test_singlet but I see that's be re-jigged :+1: